primitives/src/locktime/absolute.rs

```diff
@@ -466,31 +480,31 @@ mod tests {
 
     #[test]
     fn satisfied_by_height() {
-        let lock_by_height = LockTime::from_consensus(750_000);
-
-        let height_same = Height::from_consensus(750_000).expect("failed to parse height");
+        let height = Height::from_consensus(750_000).expect("failed to parse height");
```

Comment:
Generally I think we just use `unwrap()` since expect message wording can be hard to agree on.  However, if you do keep the expect message, I think it should match what you did with previous messages where you write what you expect to be valid.

Reply:
Yeah true, thanks.  I have kept them as expect since I find the expect messages useful for the time values and have now changed all of them to be uniform.